spec/models/landing_page/block/block_error_spec.rb

```diff
@@ -1,4 +1,6 @@
 RSpec.describe LandingPage::Block::BlockError do
+  it_behaves_like "it is a landing-page block"
```

Comment:
I think these tests should be added for the empty models as well.

Reply:
Ah yes, done.